lib/suspenders/version.rb

```diff
@@ -1,8 +1,9 @@
 module Suspenders
   RAILS_VERSION = "~> 7.0.0".freeze
-  RUBY_VERSION = IO
```

Comment:
I'm missing something: we're no longer setting `RUBY_VERSION`, but we still reference it from elsewhere (e.g. `app_builder.rb`).

Reply:
@mike-burns Yeah, that can be confusing. It turns out `RUBY_VERSION` is a global/default Ruby constant, and we were overriding it. It says which Ruby version is running. The idea is to use the global constant to set up generated applications, as long as it is within the allowed range. Does that make sense?

Reply:
Ohh, that's what I was missing. Thanks for the reminder -- yes, `RUBY_VERSION`!